core/services/relay/evm/read/method.go

```diff
@@ -165,17 +203,19 @@ func (b *MethodBinding) Unregister(_ context.Context) error { return nil }
 func (b *MethodBinding) blockNumberFromConfidence(ctx context.Context, confidenceLevel primitives.ConfidenceLevel) (*big.Int, error) {
 	confirmations, err := confidenceToConfirmations(b.confirmationsMapping, confidenceLevel)
 	if err != nil {
-		err = fmt.Errorf("%w for contract: %s, method: %s", err, b.contractName, b.method)
+		err = fmt.Errorf("%w: contract: %s; method: %s;", err, b.contractName, b.method)
 		if confidenceLevel == primitives.Unconfirmed {
-			b.lggr.Errorf("%v, now falling back to default contract call behaviour that calls latest state", err)
+			b.lggr.Debugf("%v, now falling back to default contract call behaviour that calls latest state", err)
```

Comment:
I guess this is pre-existing, but why not:
```suggestion
			b.lggr.Debugw("Falling back to default contract call behaviour that calls latest state", "err", err)
```
Actually, the error is not returned from this block, so this could be:
```suggestion
			b.lggr.Debugw("Falling back to default contract call behaviour that calls latest state", "contract", b.contractName, "method", b.method", "err", err)
```
With the `fmt.Errorf` construction moved down to the return on line 213.